receiver/podmanreceiver/factory.go

```diff
@@ -52,17 +53,53 @@ func createDefaultReceiverConfig() config.Receiver {
 	return createDefaultConfig()
 }
 
+func createReceiver(
+	ctx context.Context,
+	params component.ReceiverCreateSettings,
+	config config.Receiver,
+) (*receiver, error) {
+	podmanConfig := config.(*Config)
+	var err error
+	r := receivers[podmanConfig]
```

Comment:
This means that there will only be one receiver instance consuming both metrics and logs, right?

Reply:
Yes, the same instance will be used for both metrics and logs. The `logsConsumer` and `metricsConsumer` of the same receiver instance will be used for both logs and metrics respectively.